docs/manual-EN/2.query-language/4.statement-syntax/3.utility-statements/describe-syntax.md

```diff
@@ -15,11 +15,11 @@ Also notice that DESCRIBE is different from SHOW. Refer [SHOW](show-syntax.md).
 Obtain information about space.
 
 ```ngql
-nebula> DESCRIBE SPACE laura_space;
+nebula> DESCRIBE SPACE nba;
 ========================================================
 | ID |        Name | Partition number | Replica Factor |
```

Comment:
Partition number -> Partition Number

Reply:
![image](https://user-images.githubusercontent.com/42762957/69509786-3b60a080-0f75-11ea-8e64-0fbc39549ff1.png)

It's what `nebula` really looks like. Anyway, I think it's a good catch. Please raise an issue to change.